flink-optimizer/src/main/java/org/apache/flink/optimizer/dataproperties/GlobalProperties.java

```diff
@@ -80,9 +83,15 @@ public void setHashPartitioned(FieldList partitionedFields) {
 		this.partitioningFields = partitionedFields;
 		this.ordering = null;
 	}
-	
 
-	public void setRangePartitioned(Ordering ordering) {
+	/**
+	 * Set the parameters for range partition.
+	 * 
+	 * @param ordering Order of the partitioned fields
+	 * @param distribution The data distribution for range partition. User can supply a customized data distribution,
+	 *                     also the data distribution can be null.  
+	 */
+	public void setRangePartitioned(Ordering ordering, DataDistribution distribution) {
```

Comment:
I know you've changed this part before, but could you add a `setRangePartitioned(Ordering ordering)` method which can be used if no `DataDistribution` is provided? Would be nicer than passing `null` everywhere.